dockers/texcmp/texcmp.js

```diff
@@ -16,18 +15,12 @@ var readFile = Q.denodeify(fs.readFile);
 var writeFile = Q.denodeify(fs.writeFile);
 var mkdir = Q.denodeify(fs.mkdir);
 
-// ignore some tests, since they contain commands not supported by LaTeX
-var blacklist = {
```

Comment:
I agree with @xymostech that we should put extraneous stuff in the query string, but I also agree with @gagern that we should keep all of the data about test cases together.  Why not have ss_data.js export a dictionary where each entry is an object with a `query` property and a `nolatex` property.  Best of both worlds.